### PR TITLE
chore: publish npm packages

### PR DESCRIPTION
**Pull request by llorllale**
* De-duplicated assets (particularly the compressed wasm file)
* Added build configuration to publish to GH packages
* Removed stale `agent-js-worker-wasm` make target

Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

**Review by fqutishat — DISMISSED** [review]

**Review by troyronda — APPROVED** [review]

File: cmd/aries-js-worker/scripts/build_assets.sh [hunk 1]
```diff
@@ -6,4 +6,10 @@
 #
 
 GOOS=js GOARCH=wasm go build -o src/aries-js-worker.wasm main.go
+gzip -f src/aries-js-worker.wasm
```
Comment (troyronda) [inline cmd/aries-js-worker/scripts/build_assets.sh]: What does the CompressionPlugin now do?

Comment (llorllale) [reply cmd/aries-js-worker/scripts/build_assets.sh]: Good question - removed

Comment (troyronda) [reply cmd/aries-js-worker/scripts/build_assets.sh]: (and I guess it wasn't possible to have a common step that uses CompressionPlugin).

Comment (llorllale) [reply cmd/aries-js-worker/scripts/build_assets.sh]: Couldn't make it work without defining an entry for webpack. Couldn't quickly figure out a sensible way for it.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -114,3 +114,15 @@ jobs:
               make agent-rest-docker
               docker tag aries-framework-go/agent-rest:latest  docker.pkg.github.com/hyperledger/aries-framework-go/agent-rest:$AGENT_IMAGE_TAG
               docker push docker.pkg.github.com/hyperledger/aries-framework-go/agent-rest:$AGENT_IMAGE_TAG
+
+      - name: Publish npm packages
```
Comment (troyronda) [inline .github/workflows/build.yml]: @fqutishat @llorllale Don't we need the logic to differentiate between snapshots and releases?

Comment (fqutishat) [reply .github/workflows/build.yml]: yes

**Comment by llorllale** [comment]
Removed stale `agent-js-worker-wasm` make target

File: cmd/aries-js-worker/package.json [hunk 3]
```diff
@@ -1,20 +1,20 @@
 {
   "name": "@hyperledger/aries-framework-go",
   "version": "0.0.1",
```
Comment (fqutishat) [inline cmd/aries-js-worker/package.json]: this version should be snapshot

File: .github/workflows/build.yml (hunk 2 above)
Comment (fqutishat) [reply .github/workflows/build.yml]: https://github.com/hyperledger/aries-framework-go/pull/1197#discussion_r376084176

File: cmd/aries-js-worker/package.json (hunk 3 above)
Comment (llorllale) [reply cmd/aries-js-worker/package.json]: Aside from that - what should be the base version?

Comment (llorllale) [reply cmd/aries-js-worker/package.json]: (should we follow the general version of aries-framework-go or should this component be versioned separately?)

Comment (troyronda) [reply cmd/aries-js-worker/package.json]: It would make sense to follow general version of framework. @llorllale

Comment (llorllale) [reply cmd/aries-js-worker/package.json]: check now

**Review by fqutishat — APPROVED** [review]

**Pull request merged**